api/dispatcher/json_test.go

```diff
@@ -45,23 +45,6 @@ var _ = Describe("JSON", func() {
 				dispatcher.Dispatch([]byte(`{"method":"fake-action","arguments":["fake-arg"]}`))
 				Expect(caller.CallAction).To(Equal(action))
 				Expect(caller.CallArgs).To(Equal([]interface{}{"fake-arg"}))
-
-				dispatcher.Dispatch([]byte(`{
-          "method":"fake-action",
```

Comment:
Since we are using another way to decoding/unmarshal json, the test was not valid, so remove it